packages/common/src/scopeSupportFacets/scopeSupportFacets.types.ts

```diff
@@ -66,6 +113,7 @@ export interface ScopeSupportFacetInfo {
 
 export enum ScopeSupportFacetLevel {
   supported,
+  supportedLegacy,
```

Comment:
were you planning to do something with this field?

Reply:
Yes it was used an should be used for every language that still has legacy support. Then we can generate documentation to that fact.

Reply:
From what I can tell you're not using it anywhere. Prob worth using it at least once if you're going to introduce it? But maybe I just missed it

Reply:
It was used for typescript before we migrated all the scopes. It's now used by python as an example.